src/main/kotlin/lotto/domain/Price.kt

```diff
@@ -0,0 +1,9 @@
+package lotto.domain
+
+enum class Price(val winningPrize: Int) {
+    FIRST(2000000000),
```

Comment:
```suggestion
enum class Price(val winningPrize: Int) {
    FIRST(2_000_000_000),
```
숫자 단위가 크다면 underscore를 넣어주면 좀 더 가독성이 올라갈 수 있습니다 😄